### PR TITLE
docs: add analytics

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR introduces analytics to the official documentation.
 
Closes: https://github.com/gnolang/gno/issues/1432
  
cc @zivkovicmilos @moul 

EDIT: We should consider using [plausible.io](https://plausible.io) - it is privacy-friendly, doesn't require user approval, and DevX is already using it for the Gno Playground.

<details><summary>Contributors' checklist...</summary>

- [x] Added new tests, or not needed, or not feasible
- [x] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [x] Updated the official documentation or not needed
- [x] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [x] Added references to related issues and PRs
- [x] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

**Comment by leohhhn** [comment]
Hmm, seems my yarn installation is messing with some files. Let me try to fix this.

File: misc/docusaurus/.yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (zivkovicmilos) [inline misc/docusaurus/.yarnrc.yml:1]: Why is this required?

**Review by zivkovicmilos — APPROVED** [review]

File: misc/docusaurus/.yarnrc.yml (hunk 1 above)
Comment (leohhhn) [reply misc/docusaurus/.yarnrc.yml:1]: This is added by the newest version of yarn, ie 4.0.2. Not sure if we should keep using it, or switch to something else.

Comment (thehowl) [reply misc/docusaurus/.yarnrc.yml:1]: @alexiscolin, I'm calling you as the JavaScripter-in-chief, keep this file or burn it with fire? :fire:

Comment (alexiscolin) [reply misc/docusaurus/.yarnrc.yml:1]: This line is important for yarn v4 to create  `node_modules` building strategy. `pnp` (another strategy - used by default now) will create plug and play system that doesn't rely on usual modules folder. That is faster but less compliant with the whole ecosystem (that can be discussed). 

This line has no impact on yarn v<4, so maybe we can keep it for now and for people who uses the v4 but propose a full change in another PR as this question is not related with as this one: DX vs analytics.

**Review by alexiscolin — APPROVED** [review]

**Comment by leohhhn** [comment]
Merging this for now to get started with analytics since we already have announcements made.
